bin/utils/errorHelpers.js

```diff
@@ -4,7 +4,7 @@
 */
 "use strict";
 
-const webpackOptionsFlag = "WEBPACK_OPTIONS";
+const { webpackOptionsFlag } = require('../constants')
```

Comment:
```suggestion
const { webpackOptionsFlag } = require("../constants");
```
This would fix your lint :)

Reply:
Ahh missed 😛